src/App.js

```diff
@@ -12,20 +12,31 @@ import './App.css';
 
 class App extends Component {
   render() {
+    const {
+      authenticated,
+      contributionPoints,
+      removeNotification,
+      increaseContributionAmount,
+      username
+    } = this.props;
     return (
       <Router>
         <div className="App">
           <NavBar
-            authenticated={this.props.authenticated}
-            contributionPoints={this.props.contributionPoints}
-            username={this.props.username}
-            onRemoveNotification={this.props.removeNotification}
-            onIncreaseContributionAmount={this.props.increaseContributionAmount}
+            authenticated={authenticated}
+            contributionPoints={contributionPoints}
+            username={username}
+            onRemoveNotification={removeNotification}
+            onIncreaseContributionAmount={increaseContributionAmount}
```

Comment:
@Moe-Shoman This is normally a great idea, but I'm about to merge in code which removes this and reduxifies `NavBar`, so to save me some merge conflict headaches, can you convert this back to having all the annoying `this.props.`s in there and use `return this.props.user.authenticated ? ... : ...` below rather than the naked `authenticated`, please?

Reply:
yeah, NP